tests/common/dualtor/tunnel_traffic_utils.py

```diff
@@ -113,7 +113,7 @@ def _disassemble_ip_tos(tos):
                 check_res.append("outer packet DSCP not same as inner packet DSCP")
             exp_queue = derive_queue_id_from_dscp(outer_dscp)
 
-            time.sleep(10)
+            time.sleep(60)
```

Comment:
Instead of putting a dead 60s sleep here, it is better to continuously check the counters every 5s for 60s.
You can leverage existing utility `wait_until`
https://github.com/Azure/sonic-mgmt/blob/1ee79c0d9a9125b4002ce140e05c4a2e318d6d35/tests/common/utilities.py#L47

Basically, put the counters collection and verification logic here into a function and keep polling that function for a max of 60s.
Example usage: https://github.com/Azure/sonic-mgmt/blob/1ee79c0d9a9125b4002ce140e05c4a2e318d6d35/tests/pc/test_po_update.py#L79